spec/models/ingest/loader_spec.rb

```diff
@@ -297,5 +297,34 @@
         end
       end
     end
+
+    context 'the converter rows contains "N/A" in a numeric field' do
+      let(:invoice_rows) do
+        double(
+          'Converter::Rows',
+          data: [
+            fake_invoice_row.merge('line_number' => '1', 'Price per Unit' => 'N/A'),
+            fake_invoice_row.merge('line_number' => '2', 'Price per Unit' => '-'),
+          ],
+          row_count: 1,
+          sheet_name: 'Invoice Sheet',
+          type: 'invoice'
+        )
+      end
+
+      it 'loads data from the converter, marking each row as invalid' do
+        loader.perform
+
+        invalid_entries = file.entries.invoices.errored
+
+        aggregate_failures do
+          expect(invalid_entries.count).to eql 2
+
+          invalid_entries.each do |entry|
+            expect(entry.validation_errors.first).to match a_hash_including('message' => 'is not a number')
```

Comment:
Not that we're expecting other errors to mess up the order, but could we say `entry.validation_errors['Price per Unit'].first` instead?